modules/clientutils.js

```diff
@@ -559,14 +559,26 @@
         this.getElementsByXPath = function getElementsByXPath(expression, scope) {
             scope = scope || this.options.scope;
             var nodes = [];
-            var a = document.evaluate(expression, scope, null, XPathResult.ORDERED_NODE_SNAPSHOT_TYPE, null);
+            var a = document.evaluate(expression, scope, this.xpathNamespaceResolver, XPathResult.ORDERED_NODE_SNAPSHOT_TYPE, null);
             for (var i = 0; i < a.snapshotLength; i++) {
                 nodes.push(a.snapshotItem(i));
             }
             return nodes;
         };
 
         /**
+         * Build the xpath namespace resolver to evaluate on document
+         *
+         * @param String        prefix   The namespace prefix
+         * @return the resolve namespace or null
+         */
+        this.xpathNamespaceResolver = function xpathNamespaceResolver(prefix) {
+          if (prefix === 'svg') {
+            return 'http://www.w3.org/2000/svg';
+          }
+        };
+
```

Comment:
@jefleponot Good point! Will make the change and test with other namespaces too. Thank you.